### PR TITLE
#168506138 User should have a profile after signing up with social links

### PR DESCRIPTION
**Pull request by tolulope-od**
### What does this PR do?
Fixes an issue where a user profile is not created when signed up via social links

#### Description of Task to be completed?
When a user signs up to the application via the social links on `/api/v1/auth/google` or `/api/v1/auth/facebook`, a user profile should be generated for them in the database. This bug-fix addresses an issue where the profile wasn't created.

#### How should this be manually tested?
After cloning this repo, `cd` into it and do the following:

```bash
# fetch this branch
$ git fetch origin  bg-fix-create-profile-on-social-auth-168506138
# install dependencies
$ npm install

# run unit tests
$ npm test

# start the development server
$ npm run server
```
Using postman, visit the routes stated above

#### Any background context you want to provide?
N/a

#### What are the relevant pivotal tracker stories?
[#168506138](https://www.pivotaltracker.com/story/show/168506138)

#### Screenshots (if appropriate)

### CODE REVIEW

File: controllers/AuthController.js [hunk 1]
```diff
@@ -153,6 +155,19 @@ export default class AuthController {
         id, isVerified, type
       } = createdUser[0];
 
+      // eslint-disable-next-line no-underscore-dangle
+      if (createdUser[0]._options.isNewRecord) {
+        const avatar = gravatar.url(email, {
```
Comment (mazma1) [inline controllers/AuthController.js:160]: One of the details we get from authentication is a user's profile image. Is it possible to work with that?

Comment (tolulope-od) [reply controllers/AuthController.js:160]: Yes, but if the user doesn't have a profile picture, gravatar could help us generate one for them

Comment (mazma1) [reply controllers/AuthController.js:160]: Line 160 doesn't look like it's generated only when there is no avatar

Comment (tolulope-od) [reply controllers/AuthController.js:160]: No, it just pulls the image from the user's email profile and if it doesn't exist it creates a generic one. That's how gravatar does it

Comment (mazma1) [reply controllers/AuthController.js:160]: Ok, thanks for the clarification

**Pull request merged**